backend/route/views.py

```diff
@@ -336,28 +350,27 @@ def post_cart(request, post_id, fid):
         post_in_folder.delete()
         return HttpResponse(status=204)
 
-@require_http_methods(["POST", "DELETE"])
+@require_http_methods(["GET"])
 def post_like(request, post_id):
     logged_user_id=request.session.get('user', None)
     if not logged_user_id:
         return HttpResponse(status=405)
     user=User.objects.get(id=logged_user_id)
-    if request.method=='POST':
-        post = Post.objects.get(id=post_id)
-        like_list = post.like_set.filter(user_id=user.id)
-        Like.objects.create(user=user, post=post)
-        return JsonResponse({'postLikeUserCount': like_list.count()}, status=201)
-    else : #delete
-        post = Post.objects.get(id=post_id)
-        post.like_set.get(user=user).delete()
-        return HttpResponse(status=200)
+    post = Post.objects.get(id=post_id)
```

Comment:
I think this line should be nested in try-catch statement because `get()` method raises ` Model.DoesNotExist` exception if there is no such object. 👍